src/person.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'nameable'
+require_relative 'decorator'
+# Public: Various methods useful for performing checking on people in a library.
+class Person < Nameable

```

Comment:
- [ ] Your **Person** class is missing the method for adding rentals as you have done in the **Book** class.

Reply:
not resolved

Reply:
@Kingstalux  It doesn't matter I went for a different approach

Reply:
```Ruby
def initialize(date, book, person)
  @person = person
  person.rentals << self
  @book = book
  book.rentals << self
  @date = date
end
```
this makes that function redundant

Reply:
Hello @Leboroz, your method is noted